listenbrainz/webserver/views/api.py

```diff
@@ -247,9 +277,9 @@ def latest_import():
         if user is None:
             raise APINotFound("Cannot find user: {user_name}".format(user_name=user_name))
         return jsonify({
-                'musicbrainz_id': user['musicbrainz_id'],
-                'latest_import': 0 if not user['latest_import'] else int(user['latest_import'].strftime('%s'))
-            })
+            'musicbrainz_id': user['musicbrainz_id'],
```

Comment:
Try to avoid unrelated pep-8 changes. It makes things harder to review.

Reply:
Sorry for that, I configured my editor to format on saves. It won't happen in the future.